src/Serilog.Sinks.Console/Serilog.Sinks.Console.xproj

```diff
@@ -0,0 +1,20 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Referring to the renamed file above, which I can't comment on directly :-) The folder structure of the project should start at the root namespace Serilog, so the ConsoleSink.cs file belongs in a `Sinks\Console` subfolder.